resources/views/projects/my-projects.blade.php

```diff
@@ -18,54 +18,73 @@
         ($user->context === 'individual' &&
             ($user->individual->isConsultant() || $user->individual->isConnector()) &&
             $user->individual->isParticipant()))
```

Comment:
This likely needs to check if the user is participating in projects.

```suggestion
            ($user->individual->isParticipant() || $user->individual->inProgressParticipatingProjects()->count())))
```